ext/AvizExt/viz/rule_extraction.jl

```diff
@@ -30,25 +30,38 @@ function ADRIA.viz.rules_scatter(
     rules::Vector{Rule{Vector{Vector},Vector{Float64}}};
     opts::Dict=Dict(),
     fig_opts::Dict=Dict(),
-    axis_opts::Dict=Dict()
-)
+    axis_opts::Dict=Dict(),
+)::Figure
     f = Figure(; fig_opts...)
     g = f[1, 1] = GridLayout()
 
     # For now we only plot conditions with two clauses
     rules = filter(r -> length(r.condition) == 2, rules)
 
     ADRIA.viz.rules_scatter!(
-        g,
+        g, rs, scenarios, clusters, rules; opts=opts, axis_opts=axis_opts
+    )
+
+    return f
+end
+function ADRIA.viz.rules_scatter(
+    rs::ResultSet,
+    scenarios::DataFrame,
+    clusters::BitVector,
+    rules::Vector{Rule{Vector{Vector},Vector{Float64}}};
+    opts::Dict=Dict(),
+    fig_opts::Dict=Dict(),
+    axis_opts::Dict=Dict(),
```

Comment:
Extra comma?

Reply:
Commas added by auto-formatter. Decided to leave things as is, because:

1. extra comma doesn't hurt
2. configuring the formatter is more pain that it's worth